### ISSUE TITLE
Revise i18n for LX Music series

### ISSUE THREAD

**Issue by 3gf8jv4dv** [lyswhut/lx-music-desktop#1933]
### 解决方案检查

- [X] 我已阅读常见问题(<https://lyswhut.github.io/lx-music-doc/desktop/faq>)，并没有找到解决方案
- [X] 我已搜索issue列表(<https://github.com/lyswhut/lx-music-desktop/issues?utf8=✓&q=>)，并没有发现类似的问题

### 预期行为

All option buttons involved use the term `Close`.

### 实际行为

The close button above the software window has a tooltip of `Close` when the mouse hovers over it, while some pop-up options buttons in the software are displayed as `Closure`.

https://github.com/lyswhut/lx-music-desktop/blob/9715db1229e3bb8caa4ecd0adbcef11ec9225165/src/lang/en-us.json#L8
https://github.com/lyswhut/lx-music-desktop/blob/9715db1229e3bb8caa4ecd0adbcef11ec9225165/src/lang/en-us.json#L653

### Lx Music 版本

2.8.1-beta.2

### 操作系统版本

Windows 11, version 24H2

### 附加信息

I guess the English language in the software is machine translated, so this happens.

I have seen a lot of software where the relevant term in the English language is `Close` instead of `Closure`.

**Comment by lyswhut** [comment]
Yes, they are from Google Translate, you can provide a PR to fix it :)

**Comment by lyswhut** [comment]
Yes, you're always welcome!

**Comment by lyswhut** [comment]
Yes, if you are free.

**Comment by lyswhut** [comment]
It is planned to be released before the Spring Festival, but the specific date has not been determined yet. It may be released together with the new project.

**Comment by lyswhut** [comment]
> These repo (both desktop/mobile) is currently stacked with a lot of issues/PRs, should we clean them up (close them)?

This will result in a large number of email notifications. Do not process them for now.

> I would like to disable the option in "Settings → Downloads" in the desktop edition when the download feature is not enabled, but I can't disable the button for the "Number of Simultaneous Downloads" option. Is there any way to do this?

Not enabling "Downloads" still allows other settings to update, they will not cause problems, they just won't take effect, so there's no need to disable them

> Is this still the current policy — not to develop HarmonyOS NEXT variants for the time being?

Yes
